pulp_ansible/tests/functional/constants.py

```diff
@@ -22,12 +22,16 @@
 
 NAMESPACE_ANSIBLE = '?namespace__name=ansible'
 
+NAMESPACE_ELASTIC = '?namespace__name=elastic'
```

Comment:
Added `elastic` because it is the fixture used in the docs examples, so to ensure docs described workflow are working it is a good idea to use the same fixture.